vFXT/msazure.py

```diff
@@ -1820,8 +1820,8 @@ def add_cluster_nodes(self, cluster, count, **options):
         # our instance addresses must always reside within the subnet
         if instance_addresses[0]: # must be defined, not None
             subnet = self.connection('network').subnets.get(self.network_resource_group, self.network, subnets[0])
-            if not Cidr(subnet.address_prefix).contains(instance_addresses[0]):
-                raise vFXTConfigurationException("Cluster addresses must reside within subnet {}".format(subnets[0]))
+        if not Cidr(self._parse_subnet_address(subnet)).contains(instance_addresses[0]):
```

Comment:
Why is this being pulled outside of the `if instance_address[0]` check?

Reply:
Error in updating the file.